### DISCUSSION TITLE
pass a search_file argument to files()

### DISCUSSION THREAD

**Discussion by ayoubelmhamdi** [ibhagwan/fzf-lua#1438] [Q&A]
is there any way to pass a filename to `fzf-lua.files()`.

Because: I found a useful usage of the `search_file` argument in `telescope.nvim` is to pass a string to search for a file inside Neovim. This helps to open files in Neovim when I misspell the filename.

```bash
FILE="$1"
PWD="$(dirname "$FILE")"

nvim -c ":lua require('telescope.builtin').find_files({ search_file = \"$(basename "$FILE")\", cwd = \"$PWD\"})"
```

`vf foo.txt`

**Comment by ibhagwan** [comment]
```lua
require'fzf-lua'.files({ query = 'foo' })
-- or
:FzfLua files query=foo
```
